pyspark/dl/optim/optimizer.py

```diff
@@ -188,6 +188,14 @@ def setvalidation(self, batch_size, val_rdd, trigger, val_method=["Top1Accuracy"
         callBigDlFunc(self.bigdl_type, "setValidation", self.value, batch_size,
                       trigger, val_rdd, val_method)
 
+    def setmodel(self, model):
```

Comment:
I checked other methods in optimizer.py, looks like the naming style is Java styple(eg, setvalidation, setcheckpoint). I use the same style to make sure the consistent style.